testsuite/tests/name_resolution/prepost_formal_subp/pp_formal.ads

```diff
@@ -0,0 +1,7 @@
+generic
+   with function P (X, Y : in out Integer) return Integer with
+        Pre => X mod 2 = 0, Post => X > Y;
+        pragma Test_Block;
+   package PP_Formal is
+      procedure Ren (X, Y : in out Integer);
```

Comment:
formatting is very strange here. Should be:
```
generic
   with function P (X, Y : in out Integer) return Integer
      with Pre => X mod 2 = 0, Post => X > Y;
   pragma Test_Block;
package PP_Formal is
   procedure Ren (X, Y : in out Integer);
end PP_Formal;
```

Reply:
Oups, should be good now.